### PR TITLE
Resolve build issues with accumulo 3.0

### PR DESCRIPTION
**Pull request by DomGarguilo**
This PR mostly removes/updates things that have been removed in accumulo 3.0

I am targeting main with this PR but am wondering if a new branch should be created for 2.1 (like we have for 2.0).

I have also included a few other small misc. improvements to this PR.

### CODE REVIEW

**Comment by ctubbsii** [comment]
If the test code can be made to work against both 2.1 and 3.0, then it'd be good to have just the one branch. If not, then we need a 2.1 branch for the 2.1-specific test code, and main can remain the branch for 3.0 test code.

**Review by ctubbsii — APPROVED** [review]
Changes look fine, but see my other comment about possible need for a 2.1 branch.

**Comment by DomGarguilo** [comment]
Looks like there might be an issue when running these changes with 2.1

I am seeing the following error when running `cingest createtable`

```
Exception in thread "main" com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:270)
	at com.google.gson.Gson.fromJson(Gson.java:1058)
	at com.google.gson.Gson.fromJson(Gson.java:1016)
	at com.google.gson.Gson.fromJson(Gson.java:959)
	at com.google.gson.Gson.fromJson(Gson.java:927)
	at org.apache.accumulo.core.lock.ServiceLockData.parse(ServiceLockData.java:232)
	at org.apache.accumulo.core.fate.zookeeper.ZooCache.getLockData(ZooCache.java:545)
	at org.apache.accumulo.core.rpc.clients.TServerClient.getTabletServerConnection(TServerClient.java:71)
	at org.apache.accumulo.core.rpc.clients.ClientServiceThriftClient.getTabletServerConnection(ClientServiceThriftClient.java:45)
	at org.apache.accumulo.core.rpc.clients.TServerClient.execute(TServerClient.java:107)
	at org.apache.accumulo.core.rpc.clients.ClientServiceThriftClient.execute(ClientServiceThriftClient.java:52)
	at org.apache.accumulo.core.clientImpl.InstanceOperationsImpl.tryToModifyProperties(InstanceOperationsImpl.java:105)
	at org.apache.accumulo.core.clientImpl.InstanceOperationsImpl.modifyProperties(InstanceOperationsImpl.java:147)
	at org.apache.accumulo.testing.continuous.CreateTable.main(CreateTable.java:65)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:395)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:259)
	... 13 more

```

Not too sure what the issue is yet, looking into it.

**Comment by DomGarguilo** [comment]
I'm not too sure what was happening with the error I commented on before but I am not unable to reproduce it.

As far as I can tell, these changes work for both 2.1 and 3.0.0-SNAPSHOT.

File: src/main/java/org/apache/accumulo/testing/randomwalk/multitable/BulkImport.java [hunk 1]
```diff
@@ -98,18 +97,8 @@ public void visit(final State state, final RandWalkEnv env, Properties props) th
     }
     log.debug("Starting {} bulk import to {}", useLegacyBulk ? "legacy" : "new", tableName);
```
Comment (DomGarguilo) [inline src/main/java/org/apache/accumulo/testing/randomwalk/multitable/BulkImport.java:98]: Now that it is no longer used in the code, I think the "legacy bulk" options should be removed before this is merged

**Pull request merged**